Cargo.toml

```diff
@@ -63,7 +63,7 @@ usbd-serial = "0.1.0"
 micromath = "1.0.0"
 
 [features]
-device-selected = []
+device-selected = ["stm32f4/stm32f405"]
```

Comment:
This will cause problems if using anything other than the F405. You should only have one mcu feature enabled per compilation.